dev/io.openliberty.accesslists.internal/bnd.bnd

```diff
@@ -0,0 +1,51 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Access Lists
+Bundle-SymbolicName: io.openliberty.accesslists.internal; singleton:=true
+Bundle-Description: Provides access list and filter function for TCP, UDP \
+and HTTP channels version ${bVersion}
+
+WS-TraceGroup: ChannelFramework
+
+# For each exported package, create (in that package) a package-info.java
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+Export-Package: \
+	io.openliberty.accesslists;version="1.0";provide:=true, \
+	com.ibm.ws.tcpchannel.internal;-split-package:=merge-last;version="1.0";provide:=true, \
+	com.ibm.ws.udpchannel.internal;-split-package:=merge-last;version="1.0";provide:=true
+
+Import-Package: \
+	com.ibm.ws.bytebuffer.internal, \
+	${defaultPackageImport}
+
+Private-Package: \
```

Comment:
I will remove the Private-Package as there are no resources folders

Reply:
String ADDRESS_EXCLUDE_LIST_INVALID = "ADDRESS_EXCLUDE_LIST_INVALID";
    /** Config has an invalid IP include list */
    String ADDRESS_INCLUDE_LIST_INVALID = "ADDRESS_INCLUDE_LIST_INVALID";
Continue to be used in the channelfw project in checkAndSetValues,

Incidentally, I do no think
    String HOST_NAME_EXCLUDE_LIST_INVALID = "HOST_NAME_EXCLUDE_LIST_INVALID";
    /** Config has an invalid hostname include list */
    String HOST_NAME_INCLUDE_LIST_INVALID = "HOST_NAME_INCLUDE_LIST_INVALID";
are used anywhere